ui/lib/types.js

```diff
@@ -1,4 +1,5 @@
 export const TREE = "TREE";
 export const BLOB = "BLOB";
 
-export const MAIN_IPC_CHANNEL = "MAIN_IPC_CHANNEL";
+export const IPC_DIALOG_SHOWOPENDIALOG = "IPC_DIALOG_SHOWOPENDIALOG";
```

Comment:
As these dictate behaviour in the native/electron wrapper, it should live there. The current dependency direction seems off, given that the native part has to import part of the UI.